source/form.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Sedona</title>
+  </head>
+  <body>
+    <!-- Шапка сайта -->
+    <header class="hedaer">
+      <h1 class="header__title title-hidden">Sedona</h1>
+      <nav class="header__navigation">
+      <ul class="navigation-list navigation-list-left">
+        <li><a class="navigation-list__link">ГЛАВНАЯ</li>
+        <li><a class="navigation-list__link">ФОТО И ВИДЕО</li>
+      <ul class="navigation-list navigation-list-right">
+        <li><a class="navigation-list__link">ФОРМА ОТЗЫВА</li>
+        <li class="navigation-list__item"><a class="navigation-list__link">HTML ACADEMY</li>
+      </ul>
+      <img class="header__logo">
+      </nav>
+    </header>
+
+    <!-- Основной контент страницы -->
+    <main>
+      <!-- Баннер страницы отзыва-->
+      <section class="section form-page-banner">
+        <h1>
+
+        </h1>
+      </section>
+
+      <section class="section review">
+        <div class="review__topic section__topic">
+          <h2 class="topic-headline">Оставьте свой отзыв</h2>
```

Comment:
```suggestion
          <h2 class="review__headline">Оставьте свой отзыв</h2>
```
Я бы не создавал так много блоков, во всяком случае сейчас. Если мы определили блок как `review`, то пока что можно элементы задавать через этот блок. 
БЭМ позволяет избавляться от дефисов и писать все представления через систему блок__элемент--модификатор